examples/deadline/Local-Zone/README.md

```diff
@@ -0,0 +1,41 @@
+# RFDK Sample Application - Local Zones
+
+If you have large asset files that your worker instances need to process, any bit of latency can have a big impact on the time your renders take. This example will walk you through setting up your workers in a local zone while leaving the rest of the render farm in standard availability zones. Currently Amazon has launched a local zone in Los Angeles that is a part of the us-west-2 region, but they have more on the way. For more information on where local zones are avialable, how to get access, and what services they provide, refer to the [AWS Local Zones about page](https://aws.amazon.com/about-aws/global-infrastructure/localzones/).
+
+---
+
+_**Note:** This application is an illustrative example to showcase some of the capabilities of the RFDK. **It is not intended to be used for production render farms**, which should be built with more consideration of the security and operational needs of the system._
+
+---
+
+## Architecture
+
+This example app assumes you're familiar with the general architecture of an RFDK render farm. If not, please refer to the [All-In-AWS-Infrastructure-Basic](../All-In-AWS-Infrastructure-Basic/README.md) example for the basics.
+
+### Components
+
+#### Network Tier
+
+The network tier sets up a [VPC](https://aws.amazon.com/vpc/), which is within a [Private Hosted Zone](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/hosted-zones-private.html). These both span across all of the standard availability zones and local zones that are used, but the NAT Gateway for the VPC is only added to the stardard zones, as it is not avaiable in any local zones at this time.
```

Comment:
Not quite correct. The VPC isn't within a Private Hosted Zone. The private hosted zone is setup within the VPC. The VPC has its own internal DNS via Route53; the Private Hosted Zone adds, essentially, a DNS namespace within the VPC's DNS into which we can & do create records for the RenderQueue.

Reply:
Updated

Reply:
mention of Route53 is removed which seems fair. There is a link to the All-in-AWS-Infrastructure-Basic example which mentioned the Private Hosted Zone. Resolving